### PR TITLE
Scroll anchoring update

### PR DESCRIPTION
**Pull request by gauntface**
cc @PaulKinlan

### CODE REVIEW

**Comment by gauntface** [comment]
@kaycebasques @jpmedley would either of you be able to give this the once over?

@PaulKinlan seemed to be ok with the article offline.

**Comment by PaulKinlan** [comment]
Content wise I am A.OK with this.  Grammar + Spolling I will leave it to better people.

**Comment by jpmedley** [comment]
At the risk of sounding immodest, I'm on it.

File: src/content/en/updates/posts/2016/04/scroll-anchoring.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
+---
+layout: updates/post
+title: "Experiment Time: Scroll Anchoring"
+description: "Scroll anchoring is a new UA intervention
+that stops a web page from causing the page to move around
+once you've started reading the content."
```
Comment (jpmedley) [inline src/content/en/updates/posts/2016/04/scroll-anchoring.md]: Doesn't description need to be a single line?

Comment (jpmedley) [inline src/content/en/updates/posts/2016/04/scroll-anchoring.md]: Also, should I already know what a UA intervention is?

File: src/content/en/updates/posts/2016/04/scroll-anchoring.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+1. Go to chrome://flags/#enable-scroll-anchoring on *Chrome Dev / Canary*
+1. Select “Enabled” from the dropdown
+1. Click “*Relaunch Now*” at the bottom of the screen
+
+With this you’ll have scroll anchoring enabled.
+
+{% ytvideo i0iitEFArxA %}
+
+We've been using this for a while and we believe that this drastically improves the experience for all users on the web but we want to make sure that it works well everywhere, if you spot any examples where scroll anchoring failed to handle reflows in the page or examples where it shouldn’t have intervened, *we desperately want to hear about it!*
+
```
Comment (jpmedley) [inline src/content/en/updates/posts/2016/04/scroll-anchoring.md]: If you spot... is the start of a new sentence.

File: src/content/en/updates/posts/2016/04/scroll-anchoring.md [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+This change alters the effect of scrolling caused by reflows. For
+example, adding a class name to an element that causes it to
+increase in height will cause a reflow and scroll anchoring will prevent the
+page from jumping around.
+
+Calling `window.scrollTo(0, 1)` (Yes the old school hack) wouldn't cause a reflow
+and will behave normally. The same goes for touch events.
+
+If you do find an example where scroll anchoring is affecting your page, please
```
Comment (jpmedley) [inline src/content/en/updates/posts/2016/04/scroll-anchoring.md]: Remove the word do.

**Comment by jpmedley** [comment]
Found a few items that need addressed. Looks good otherwise.

File: src/content/en/updates/posts/2016/04/scroll-anchoring.md (hunk 1 above)
Comment (dtharmen) [inline src/content/en/updates/posts/2016/04/scroll-anchoring.md]: 6 April 2016

**Comment by gauntface** [comment]
@jpmedley I made the changes you highlighted.

**Comment by PaulKinlan** [comment]
@jpmedley are we good?

**Pull request merged**